docker-compose/tail-sampling/README.md

```diff
@@ -0,0 +1,37 @@
+# Tail-Based Sampling Processor
+
+This `docker compose` environment provides a sample configuration of a Jaeger collector utilizing the
+[Tail-Based Sampling Processor in OpenTelemtry](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/tailsamplingprocessor/README.md).
+
+## Description of Setup
+
+The `docker-compose.yml` contains three services and their functions are outlined as follows:
+
+1. `jaeger` - This is the Jaeger V2 collector that samples traces using the `tail_sampling` processor.
+The configuration for this service is in [jaeger-v2-config.yml](./jaeger-v2-config.yml).
+The `tail_sampling` processor has one policy that only captures traces from the services `tracegen-02` and `tracegen-04`.
+For a full list of policies that can be added to the `tail_sampling` processor, check out [this README](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/tailsamplingprocessor/README.md).
+2. `otel_collector` - This is an OpenTelemtry collector with a `loadbalancing` exporter that routes requests to `jaeger`.
+The configuration for this service is in [otel-collector-config-connector.yml](./otel-collector-config-connector.yml).
+The purpose of this collector is to collect spans from different services and forward all spans with the same `traceID`
+to the same downstream collector instance (`jaeger` in this case), so that sampling decisions for a given trace can be
+made in the same collector instance.
```

Comment:
```suggestion
made in the same collector instance. Strictly speaking it's not needed in this setup because there is only one instance of Jaeger collector, but in a real world deployment this load balancing step is required and we include it for completeness of the example.
```